tests/unit-ngv/test_astrocyte.py

```diff
@@ -1,55 +1,67 @@
-import pytest
-
-from neurodamus import Neurodamus
+import numpy.testing as npt
+from neurodamus.ngv import Astrocyte
 from tests.conftest import NGV_DIR
+from unittest.mock import MagicMock
+
+meinfos = MagicMock()
```

Comment:
how hard is it to create real MEInfos/circuit_conf as opposed to using MagicMock?
In my experience, using mocks like this is fragile, as whatever is being mocked, which isn't indicated here, may change during a refactor, but than this code will still pass the unit tests.

Reply:
ok, I can try. I hope it is not entangled too much in the rest of the code. If I need to use the whole neurodamus then it becomes an integration test

Reply:
magicMock removed